### PR TITLE
Add code for figure generation

### PR DESCRIPTION
**Pull request by ngreenwald**
Just need to update readme with link to zip file

### CODE REVIEW

**Comment by ngreenwald** [comment]
Not sure what level of polish is necessary for a preprint?

File: 2021-Greenwald_Miller_et_al-Mesmer/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This repo contains the code to reproduce the figures from Greenwald, Miller et al.
+
+Some plots require access to the TissueNet dataset, and are currently commented out. Once the dataset is made available we will 
+update the corresponding sections of the code.
+
+The data to generate the figures can be found in [this google bucket](https://storage.googleapis.com/publications-data/mesmer-preprint/mesmer_publication_data.zip): please download it and place it in the top-level directory for the example code to run correctly.
```
Comment (willgraf) [inline 2021-Greenwald_Miller_et_al-Mesmer/README.md]: No need to tell them its a google bucket

```suggestion
The data to generate the figures can be found [here](https://storage.googleapis.com/publications-data/mesmer-preprint/mesmer_publication_data.zip): please download it and place it in the top-level directory for the example code to run correctly.
```

File: 2021-Greenwald_Miller_et_al-Mesmer/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+This repo contains the code to reproduce the figures from Greenwald, Miller et al.
+
+Some plots require access to the TissueNet dataset, and are currently commented out. Once the dataset is made available we will 
+update the corresponding sections of the code.
```
Comment (willgraf) [inline 2021-Greenwald_Miller_et_al-Mesmer/README.md]: Fix weird whitespace.

```suggestion
Some plots require access to the TissueNet dataset, and are currently commented out. Once the dataset is made available we will update the corresponding sections of the code.
```

File: 2021-Greenwald_Miller_et_al-Mesmer/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+git+https://github.com/angelolab/ark-analysis.git@6b21140e099ce21a458699573e8dbda9127c3d03
```
Comment (willgraf) [inline 2021-Greenwald_Miller_et_al-Mesmer/requirements.txt]: eww

**Review by willgraf — CHANGES_REQUESTED** [review]
A few README changes. I haven't reviewed the figure scripts - if the figures are already made I don't see the need for changes.

File: 2021-Greenwald_Miller_et_al-Mesmer/requirements.txt (hunk 3 above)
Comment (ngreenwald) [reply 2021-Greenwald_Miller_et_al-Mesmer/requirements.txt]: Hahahaha, avert your eyes! You weren't supposed to see this. I'll make a new release this weekend.

**Review by willgraf — APPROVED** [review]

**Pull request merged**